src/app/history/history-bet/history-bet-item/history-bet-item.component.html

```diff
@@ -0,0 +1,5 @@
+<div class="history-item" [ngClass]="{ odd: index % 2 !== 0 }">
```

Comment:
odd here and odd on table.. as I said - row as a component is an overhead in that use-case, but it is not a critical issue

Reply:
I've changed it